proto/nodeinstance/info/v1/info.proto

```diff
@@ -83,4 +85,17 @@ message NodeInfo {
   reserved 20;
 
   map<string, string> host_labels = 21;
+
+  MachineLearningInfo MLInfo = 22;
+}
+
+message MachineLearningInfo {
+  // true when UpdateNodeInfo.child == true && parent runs ML for child
+  bool ml_on_parent = 1;
+
+  // child has ML capability
```

Comment:
Since this about a NodeInstance, then it's not necessarily a child or a parent, but about the Node or NodeInstance itself